client/cli/src/commands/run_cmd.rs

```diff
@@ -390,6 +394,10 @@ impl CliConfiguration for RunCmd {
 		})
 	}
 
+	fn runner_port(&self, _default_listen_port: u16) -> Result<Option<u16>> {
+		Ok(self.runner_port.clone())
```

Comment:
We should set the default value here if `runner_port` was not provided in cli param